packages/cli/src/cmds/compare/ChangeReporter.ts

```diff
@@ -489,6 +495,36 @@ export class ReportFieldCalculator {
     };
   }
 
+  async formatEndpoint(input: string) {
+    if (input.startsWith('paths.')) {
+      const parts = input.split('.');
+
+      // Check if the input only has the 'paths.' and endpoint
+      if (parts.length === 2) {
+        return parts[1].replace('.', '/');
```

Comment:
Hi @moxw, thank you for your PR! Out of curiosity, what's the condition in which you need to replace `.` with `/`?

Reply:
Thank you for your review, @dustinbyrne. Upon further examination, the replacement of '.' with '/' was unnecessary in this context. I have removed it in the latest commit.

Reply:
@dustinbyrne do you have anything else you'd like addressed in this before we approve it?

Reply:
Nope! This change looks to be well tested so I've approved 👍